DESIGN.md

```diff
@@ -162,18 +162,23 @@ import h5py
 import numpy as np
 ```
 
-Optionally in the script preamble we can add some metadata to our dataset, such as a citation pointing to an upstream source, a dataset description, a web link, code licence, and version number. These values will be folded via the `DatasetInfo` method in the `_info` function of our dataloader.
+In the script preamble we add some metadata to our dataset, such as a citation pointing to an upstream source, a dataset description, a web link, code licence, and version number. These values will be folded via the `DatasetInfo` method in the `_info` function of our dataloader, and are used to quickly access citation and acknowledgement information as outlined in the [Citations & Acknowledgements](README.md#citations--acknowledgements) section of the README. **Note:** An initial summary of what information is missing from each dataset can be checked with the [info.py](scripts/info.py) script as `python scripts/info.py --missing`.
 
 ```python
-_CITATION = """\
+_CITATION = r"""% CITATION
 @InProceedings{huggingface:dataset,
-title = {A great new dataset},
-author={huggingface, Inc.
-},
-year={2020}
+    title = {A great new dataset},
+    author={huggingface, Inc.
+    },
+    year={2020}
 }
 """
```

Comment:
Is the citation supposed to point to a HF publication or to the original paper?

Reply:
To the paper that should be cited if the dataset is used.